core/modules/entity/entity.class.inc

```diff
@@ -177,13 +177,40 @@ interface EntityInterface {
    * @since 1.13.0 Method added.
    */
   public function getFieldValues($field_name, $value_key = 'value', $langcode = NULL);
+
+  /**
+   * Returns the revision identifier of the entity.
+   *
+   * @return
+   *   The revision identifier of the entity, or NULL if the entity does not
+   *   have a revision identifier.
+   */
+  public function getRevisionId();
+
+  /**
+   * Checks if this entity is the default revision.
+   *
+   * @param bool $new_value
+   *   (optional) A Boolean to (re)set the is_default_revision flag.
+   *
+   * @return bool
+   *   TRUE if the entity is the default revision, FALSE otherwise.
+   */
+  public function isDefaultRevision($new_value = NULL);
```

Comment:
I think separating the parameter for `$new_value` into a different method would be more clear. e.g. add a `setDefaultRevision()` in addition to `isDefaultRevision()`.